src/test/java/model/data/CombinedDataTableTest.java

```diff
@@ -0,0 +1,137 @@
+package model.data;
+
+import model.data.value.DataValue;
+import model.data.value.StringValue;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.*;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by jens on 5/18/15.
+ */
+public class CombinedDataTableTest {
+	private List<DataRow> rows;
+	private DataColumn[] columns;
+	private List<DataTable> dataTables;
```

Comment:
Lijkt mij inderdaad mooier hier een DataModel of DataTable te gerbuiken i.p.v. zoiets handmatig te simuleren